mlrun/execution.py

```diff
@@ -587,9 +713,7 @@ def log_model(
         """
 
         if training_set is not None and inputs:
-            raise mlrun.errors.MLRunInvalidArgumentError(
-                "cannot specify inputs and training set together"
-            )
+            raise MLCtxValueError("cannot specify inputs and training set together")
```

Comment:
What's the idea behind this `MLCtxValueError` ? 
It doesn't even inherit from `ValueError`... I don't see a reason not using `MLRunInvalidArgumentError` everywhere

Reply:
This is what we had before, I'm not a fan of sticking MLRunInvalidArgumentError everywhere, why don't we just use the built in ValueError?